doc/community/extensions.rst

```diff
@@ -108,6 +108,14 @@ Haskell
 * :flyc:`flycheck-haskell` (*official*) configures Flycheck from the Cabal
   settings and sandbox in Haskell projects.
 
+Kotlin
+------
+
+* :gh:`whirm/flycheck-kotlin` adds a Kotlin syntax checker using ``ktlint``
```

Comment:
The markup here seems wrong - e.g. it should be Kotlin_ and ktlint_ for the links to work.

Reply:
Thanks. I had another look at how the other sections were doing it, and I've updated it now to make the links work. Please have another look :)